src/app/components/project-map/new-template-dialog/new-template-dialog.component.ts

```diff
@@ -662,7 +632,6 @@ export class NewTemplateDialogComponent implements OnInit {
     qemuTemplate.first_port_name = this.applianceToInstall.first_port_name;
```

Comment:
Here we need to add `qemuTemplate.platform = this.applianceToInstall.qemu.arch;` so the Qemu template can be configured with the right platform (the field is named `arch` in the template file)